src/questions/QuestionDB.java

```diff
@@ -0,0 +1,345 @@
+package questions;
+
+import model.Question;
+import model.Category;
+
+/**
+ * @author Roger
+ */
+
+import java.sql.*;
+import java.util.ArrayList;
+
+public class QuestionDB implements QuestionStore {
+
+	private Connection connection = null;
+	private PreparedStatement preparedStatement = null;
+	private ResultSet resultSet = null;
+	
+
+	public QuestionDB(Connection connection) {
+
+		this.setConnection(connection);
+	}
+
+	public Connection getConnection() {
+		return connection;
+	}
+
+	private void setConnection(Connection connection) {
+		this.connection = connection;
```

Comment:
Auf die Frage ob das noch gebraucht wird - Ja darüber wird nachher in den Methoden der jeweilige Query ausgeführt -> preparedStatement = connection.prepareStatement(query);

Oder habe ich euch falsch verstanden? ^^

Reply:
Die Frage war auf den auskommentierten Block bezogen. :D

see: https://github.com/MninaTB/AE_Quiz_Projekt24/pull/61#discussion_r640345421

Reply:
Haha oh ok, dann nicht xD